autoload/tagbar.vim

```diff
@@ -1397,6 +1397,12 @@ function! s:ExecuteCtagsOnFile(fname, realfname, typeinfo) abort
             let ctags_args += [ '-V' ]
         endif
 
+        " Define a new language before adding --language-force to the list of
+        " arguments
+        if has_key(a:typeinfo, 'deflang')
+            let ctags_args += ['--langdef=' . expand(a:typeinfo.deflang)]
+        endif
```

Comment:
We might be able to just key off the `deffile` instead. Then there wouldn't need to be any new config keys. Can you give this a try and see if this works?
```suggestion
        if has_key(a:typeinfo, 'deffile')
            let ctags_args += ['--langdef=' . a:typeinfo.ctagstype]
        endif
```